index.js

```diff
@@ -203,6 +205,7 @@ function connect(params, cb) {
 	assert.optionalBool(params.emptyChar2null, "params.emptyChar2null");
 	assert.optionalBool(params.long2number, "params.long2number");
 	assert.optionalString(params.unixSocket, "params.unixSocket");
+	assert.optionalBool(params.useTLS, "params.useTLS");
```

Comment:
Could you document the new `useTLS ` param here https://github.com/michaelwittig/node-q#connectparams-cb ?